README.md

```diff
@@ -1,2 +1,49 @@
-# fxa-geodb
-Firefox Accounts IP geolocation library
+## Firefox Accounts GeoDB Repo for Geolocation based services
+Provides a wrapper around [node-maxmind] (https://github.com/runk/node-maxmind) for getting the location data in an easy-to-read format.
+
+### Getting started
+Clone/fork the repo and run `npm i`. Then, include the module in your source file, like so:
+
+```JavaScript
+var geoDb = require('./fxa-geodb');
+```
+--
+### API
+The function returns a promise that may either resolve (on successful finding of location data) or reject (if either the ip was invalid, or location data could not be found). Call the function, like so:
+
+```JavaScript
+geoDb(ip).then(function (location) {
+        // success, resolved
+        // location data is available here
+      }, function (err) {
+        // rejected :(
+        // Uh-oh error
+      });
+```
+
+On successful resolution of the promise, the `location` object has the following data:
+
+```JavaScript
+accuracy: 'accuracy-radius-in-km', // 5
+city: 'human-readable-city-name', // Mountain View
+continent: 'human-readable-continent-name', // North America
+country: 'human-readable-country-name', // USA
+ll: {
+    latitude: 'latitude-in-decimal', // 37.386
+    longitude: 'longitude-in-decimal' // -122.0838
+},
+time_zone: 'IANA-compatible-timezone', // America/Los_Angeles
+```
+--
+
+### Testing
+Mocha Tests are provided inside `test/fxa-geodb.js`. To run the tests, simply call `npm test`.
```

Comment:
`provided inside` => `located in the test subdirectory`. If you add more tests for the cron function, you'll have two test files, so might as well just put `test`.

Can you remove `simply` from `simply call` here and below?